src/reactapp/src/context/Cart/order/actions.js

```diff
@@ -25,7 +25,7 @@ async function verifyPaymentMethod(dispatch, values, selectedPaymentMethod) {
     const paymentMethod = _get(values, PAYMENT_METHOD_FORM);
 
     if (!paymentMethod.code) {
-      throw new Error('Pyament method not available');
+      throw new Error('Payment method not available');
```

Comment:
Good catch. Add translation here please. Also apply translation to all messages in this PR. Dont forget to add this in the file `src/reactapp/public/i18n/en_US.json`

Reply:
DOne

Reply:
Can we add this to string `src/reactapp/public/i18n/en_US.json` please. There are many other translation strings in this commit that we need to add into `en_US.json`